com.unity.render-pipelines.universal/ShaderLibrary/RealtimeLights.hlsl

```diff
@@ -266,6 +275,10 @@ Light GetAdditionalLight(uint i, float3 positionWS, half4 shadowMask)
     half4 occlusionProbeChannels = _AdditionalLightsOcclusionProbes[perObjectLightIndex];
 #endif
     light.shadowAttenuation = AdditionalLightShadow(perObjectLightIndex, positionWS, light.direction, shadowMask, occlusionProbeChannels);
+#ifdef _ADDITIONAL_LIGHT_COOKIES
+    half3 cookieColor = URP_LightCookie_SampleAdditionalLightCookie(perObjectLightIndex, positionWS /*, TODO: light type*/);
```

Comment:
This line will introduce a shader warning here when real is defined as float since the function returns a real3.

Reply:
Thanks. I will fix it.